src/coreclr/src/jit/lsra.cpp

```diff
@@ -9443,7 +9443,7 @@ void LinearScan::lsraGetOperandString(GenTree*          tree,
                                       unsigned          operandStringLength)
 {
     const char* lastUseChar = "";
-    if ((tree->gtFlags & GTF_VAR_DEATH) != 0)
+    if (tree->OperIsScalarLocal() && ((tree->gtFlags & GTF_VAR_DEATH) != 0))
```

Comment:
This is just a fix to the dump - we shouldn't be checking last use on anything but lclVars.